src/core/i18n/translation/locales/en.ts

```diff
@@ -158,7 +160,19 @@ export const translation: ITranslations = {
             changePinSubtitle: 'Your current PIN is required in order to change it',
             recoverWalletPinSubtitle: 'Your PIN is required\nto recover your Moonlet wallet',
             connectHardwareWalletPinSubtitle:
-                'Your PIN is required\nto connect your hardware wallet'
+                'Your PIN is required\nto connect your hardware wallet',
+            moonletDisabled: 'Moonlet is disabled',
+            disabledDetails: 'try again in {{duration}} {{measurement}}'
+        },
+        Time: {
+            second: 'second',
```

Comment:
are all these actually used?

Reply:
Yes, all of them are used
![image](https://user-images.githubusercontent.com/32679726/77957828-c8798480-72dc-11ea-8379-b4ed652d2124.png)